### PR TITLE
fix: Kong for Kubernetes Enterprise

### PR DESCRIPTION
**Pull request by Rajakavitha1**
### Description

validated and rewrote the instructions.

Here are the terminal outputs of the instructions in the deploy preview.
 

[k4k8s-enterprise-yaml.txt](https://github.co
[k4k8senterprisekustomise.txt](https://github.com/Kong/docs.konghq.com/files/12829238/k4k8senterprisekustomise.txt)
m/Kong/docs.kon
[k4k8senterprizehelm.txt](https://github.com/Kong/docs.konghq.com/files/12829242/k4k8senterprizehelm.txt)
ghq.com/files/12829234/k4k8s-enterprise-yaml.txt)
[k4k8s-enterprise-yaml.txt](https://github.com/Kong/docs.konghq.com/files/12829255/k4k8s-enterprise-yaml.txt)

### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md [hunk 1]
```diff
@@ -6,128 +6,181 @@ Kong for Kubernetes Enterprise is an enhanced version of
...
+1. Create the `kong` namespace in the cluster to create a secret.
+
+    ```bash
+    $ kubectl create namespace kong
+    ```
+    The results should look like this:
+    ```text
+    namespace/kong created
+    ```
+1. Create [Kong Enterprise License secret](#kong-enterprise-license-secret)
```
Comment (mheap) [inline app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md:22]: Why does this link further down the page? I'd expect it to be inline

File: app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md [hunk 2]
```diff
@@ -6,128 +6,181 @@ Kong for Kubernetes Enterprise is an enhanced version of
...
+
+1. Create a `values.yaml` file with these details to deploy using Helm charts.
+    ```
+    image:
+      repository: kong/kong-gateway
+      tag: 2.2.1.0-alpine
```
Comment (mheap) [inline app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md:28]:
```suggestion
      tag: {{ site.data.kong_latest_gateway.ee-version }}
```

File: app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md [hunk 3]
```diff
@@ -6,128 +6,181 @@ Kong for Kubernetes Enterprise is an enhanced version of
...
+    ```bash
+    $ kubectl create namespace kong
+    ```
+    The results should look like this:
+    ```text
+    namespace/kong created
+    ```
+1. Create [Kong Enterprise License secret](#kong-enterprise-license-secret)
+
+1. Create a `values.yaml` file with these details to deploy using Helm charts.
```
Comment (mheap) [inline app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md:24]: Why do I need a `values.yaml` if I'm not using Helm?

File: app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md [hunk 4]
```diff
@@ -6,128 +6,181 @@ Kong for Kubernetes Enterprise is an enhanced version of
...
+{% endnavtab %}
+{% navtab Helm %}
+You can use Helm to install Kong using the official Helm chart:
+
+1. Install {{site.kic_product_name}} and {{ site.base_gateway }} with Helm:
+
+    ```
+    $ helm repo add kong https://charts.konghq.com
+    $ helm repo update
+    $ helm install kong kong/ingress -n kong --create-namespace
```
Comment (mheap) [inline app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md:162]:
```suggestion
    $ helm install kong kong/ingress -n kong --create-namespace --values ./values.yaml
```

**Review by mheap — CHANGES_REQUESTED** [review]
Please review and ensure that the guide only shows relevant information at all times.

All installation methods also need testing to ensure that the license is applied as expected. To check the license, try applying a rate-limit-advanced plugin. 

https://docs.konghq.com/hub/kong-inc/rate-limiting-advanced/how-to/basic-example/?tab=enable-globally,kubernetes

**Comment by mheap** [comment]
I force pushed after some major changes. The files are now in `_src/kic-v2`.

```
git checkout k4k8s-enterprise-rajie
git fetch origin
git reset --hard origin/k4k8s-enterprise-rajie
```

**Comment by Rajakavitha1** [comment]
> [k4k8s-enterprise-yaml.txt](https://github.co [k4k8senterprisekustomise.txt](https://github.com/Kong/docs.konghq.com/files/12829238/k4k8senterprisekustomise.txt) m/Kong/docs.kon [k4k8senterprizehelm.txt](https://github.com/Kong/docs.konghq.com/files/12829242/k4k8senterprizehelm.txt) ghq.com/files/12829234/k4k8s-enterprise-yaml.txt) [k4k8s-enterprise-yaml.txt](https://github.com/Kong/docs.konghq.com/files/12829255/k4k8s-enterprise-yaml.txt)

Hi @mheap . All the install methods were tested.
[k4k8s-enterprise-yaml.txt](https://github.co/
[k4k8senterprisekustomise.txt](https://github.com/Kong/docs.konghq.com/files/12829238/k4k8senterprisekustomise.txt)
m/Kong/docs.kon
[k4k8senterprizehelm.txt](https://github.com/Kong/docs.konghq.com/files/12829242/k4k8senterprizehelm.txt)
ghq.com/files/12829234/[k4k8s-enterprise-yaml.txt](https://github.com/Kong/docs.konghq.com/files/12829255/k4k8s-enterprise-yaml.txt))
k4k8s-enterprise-yaml.txt

File: app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md (hunk 2 above)
Comment (Rajakavitha1) [reply app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md]: fixed

File: app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md (hunk 1 above)
Comment (Rajakavitha1) [reply app/_src/kubernetes-ingress-controller/deployment/k4k8s-enterprise.md]: fixed

File: app/_src/kic-v2/deployment/k4k8s-enterprise.md [hunk 5]
```diff
@@ -6,128 +6,179 @@ Kong for Kubernetes Enterprise is an enhanced version of
...
+    ```
+    image:
+      repository: kong/kong-gateway
+      tag: {{ site.data.kong_latest_gateway.ee-version }}
+    env:
+      LICENSE_DATA:
+        valueFrom:
+          secretKeyRef:
+            name: kong-enterprise-license
+            key: license
```
Comment (mheap) [inline app/_src/kic-v2/deployment/k4k8s-enterprise.md:153]:
```suggestion
    gateway:
        image:
          repository: kong/kong-gateway
          tag: {{ site.data.kong_latest_gateway.ee-version }}
        env:
          LICENSE_DATA:
            valueFrom:
              secretKeyRef:
                name: kong-enterprise-license
                key: license
```

**Review by mheap — CHANGES_REQUESTED** [review]

**Review by mheap — CHANGES_REQUESTED** [review]
Please fix the Vale issues

**Review by mheap — APPROVED** [review]

**Pull request merged**